echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/config/SchedulerConfiguration.groovy

```diff
@@ -67,6 +68,18 @@ class SchedulerConfiguration {
     return factoryBean
   }
 
+  @Bean
+  /**
+   * Job factory used to create jobs as beans on behalf of Quartz
+   */
+  JobFactory jobFactory(ApplicationContext applicationContext)
+  {
```

Comment:
We also don't newline opening squigglies.

Reply:
dunno what happened there